capstone/src/main/webapp/script.js

```diff
@@ -47,5 +48,97 @@ function  formatCardInHTML(studySet) {
           <span> By  ${studySet.user_author} from ${studySet.university}</span>
          </div>
       </div>
-    </a>`;
+   </a>`;
+}
+
+//View Study Set
+async function loadStudySetToPage() {
+  var urlParams = new URLSearchParams(window.location.search);
+  const studySetId = urlParams.get("id");
+  const response = await fetch("/study_set/" + studySetId);
+  const studySet = await response.json();
+
+  document.getElementById(
+    "studySetDirectory-container"
```

Comment:
IDs should use hyphens rather than camelcase: https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Name_Delimiters